packages/react/src/avatar/avatar.tsx

```diff
@@ -0,0 +1,13 @@
+type AvatarProps = {
+  src: string;
```

Comment:
Vi må håndtere fallback i komponenten. Det er ikke nødvendigvis slik at man alltid har et bilde å vise.

Kan vi vise user-ikonet vårt feks?

Vi burde faktisk håndtere onErrror eventet for bildet og vise fallbacken automatisk også dersom bildet ikke laster.